### DISCUSSION TITLE
Add NSFW blur for vulgar messages

### DISCUSSION THREAD

**Discussion by WilliamDormer** [PullDating/Org#132] [Ideas & Features]
A lot of women dislike the apps because they get abusive/harassing/overly sexual messages without wanting them. Maybe we could detect these kinds of messages and blur them out to prevent these negative interactions.

**Comment by ajbott** [comment]
I think this would be super hard to get right. I think having a satisfying reporting mechanic to punish users that send these vulgar messages would be more satisfying than blurring out the message. But with that being said, this would make for a good side feature for users that don't wish to see vulgar messages in the first place.
